### ISSUE TITLE
Item note/label

### ISSUE THREAD

**Issue by cbroeren** [cbroeren/ember-context-menu#10]

**Comment by rwwagner90** [comment]
@cbroeren can you explain this in more detail?

**Comment by cbroeren** [comment]
Yes. I had something in mind to have a menu item label/description added. Which might look something like

**Item name**
_Item description_

**Do something**
_This inverses selection_
